src/App.tsx

```diff
@@ -1,10 +1,54 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useMemo, useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
+import { FilterBy } from './types/FilterBy';
+import { warningTimer } from './utils/warningTimer';
+import { TodoInfo } from './components/TodoInfo';
+import { Footer } from './components/Footer/Footer';
+import { Header } from './components/Header';
+import { Notification } from './components/Notification';
 
-const USER_ID = 0;
+const USER_ID = 6316;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.ALL);
+  const [hasError, setHasError] = useState(false);
+  const hasActiveTodos = todos.filter(todo => !todo.completed);
+  const hasCompletedTodos = todos.some(todo => todo.completed === true);
+
+  useEffect(() => {
+    const onLoadGetTodos = async () => {
+      try {
+        const todosData = await getTodos(USER_ID);
+
+        setTodos(todosData);
+      } catch (error) {
+        setHasError(true);
+        warningTimer(setHasError, false, 3000);
+      }
+    };
+
+    onLoadGetTodos();
+  }, []);
+
+  const visibleTodos = useMemo(() => {
+    const filteredTodos = todos.filter(todo => {
```

Comment:
I guess, it is possible to create a separate function for this part of code

Reply:
let us do it